client/src/components/Offer/OfferDetails/OfferDetails.tsx

```diff
@@ -72,3 +61,19 @@ export const OfferDetails = ({
     </div>
   );
 };
+
+type Salary = {
+  from: number;
+  to: number;
+  currency: string;
+};
+
+export const getNiceSalary = (salary: Salary) => {
+  if (salary?.from && salary?.to) {
+    return `${salary.currency || ""} ${salary.from} - ${salary.to}`;
+  }
+
+  if (salary) {
+    return "";
+  }
+};
```

Comment:
You meant something like this probably:

```suggestion
if (!salary) {
  return "";
}

    return `${salary.currency || ""} ${salary.from} - ${salary.to}`;
};
```